android/app/build.gradle

```diff
@@ -189,6 +189,11 @@ android {
 }
 
 dependencies {
+    if (targetSdkVersion >= 31) {
+        // This is to fix a known issue in Android 31 (API 12)
```

Comment:
Were you able to reproduce this issue in 32 and 33?

Reply:
Yep. It was a bit unexpected, but it might have to be related to the versions in the SDK. Originally I had `targetSdkVersion == 31` but when I bumped the project target, I got the `FLAG_MUTABLE` issue too. I did not want to go into that rabbit hole knowing that all this comes from some Android bug.